hare/algorithm.go

```diff
@@ -739,6 +730,11 @@ func (proc *consensusProcess) processCommitMsg(ctx context.Context, msg *Msg) {
 }
 
 func (proc *consensusProcess) processNotifyMsg(ctx context.Context, msg *Msg) {
+	if proc.notifyTracker == nil {
+		proc.WithContext(ctx).With().Error("notify tracker is nil")
```

Comment:
`With()` is redundant here.
```suggestion
		proc.WithContext(ctx).Error("notify tracker is nil")
```